priv/gettext/es/LC_MESSAGES/dotcom.po

```diff
@@ -598,7 +598,7 @@ msgid "Typically fewer than 33% of seats available and distancing is unlikely (~
 msgstr "Típicamente menos del 33% de los asientos disponibles y el distanciamiento es poco probable (~3+ personas por fila)"
 
 msgid "Main Hotline"
-msgstr "Línea caliente principal"
+msgstr "Hotline principal"
```

Comment:
ah, did we run into a using-different-versions-of-libretranslate problem again? 😬